sdk/python/pulumi_aws/rds/cluster_instance.py

```diff
@@ -64,6 +64,7 @@ class ClusterInstance(pulumi.CustomResource):
     identifier: pulumi.Output[str]
     """
     The indentifier for the RDS instance, if omitted, this provider will assign a random, unique identifier.
+    >>>>>>> Update terminology for use in Pulumi documentation
```

Comment:
Where are these coming from...?

Reply:
And that, my friend is a dodgy merge conflict fix and why reviews are awesome - will push a fix when I get to the airport and search for any more occurances